lib/house.rb

```diff
@@ -0,0 +1,55 @@
+require 'pry'
+
+class House
+    attr_reader :price, :address, :rooms
+    def initialize (price, address)
+        @price = price.delete("$").to_i
+        @address = address
+        @rooms = []
+    end
+
+    def add_room(room)
+        @rooms << room
+    end
+
+    def above_market_average?
+        if @price > 500000
+            true
+        else
+            false
+        end
```

Comment:
Since `@price > 500000` will evaluate as `true` or `false` we can shrink this whole method down to just that one line.
```suggestion
        @price > 500000
```